docs/ReleaseNotes.md

```diff
@@ -7,6 +7,9 @@ Published as version 14.18.0
 
 New Features:
 * Updated pseudo strings to be encapsulated with square brackets so that you can tell when the resource string starts and ends.
+* Added more pseudomap data to test by adding pseudoLocale.
+    * Added tall properly in some `pseudomap.json` files which is the tallest glyphs in the script
+      and it is going to be included first in a pseudo string.
```

Comment:
* Added more pseudomap data for use in testing different fonts using pseudo locales.
    * Added a "tall" property in some `pseudomap.json` files which gives the tallest glyph in the script.
      This tallest glyph is included at the beginning of a pseudo localized string in order to test
      if there is enough room for the height of text in each UI element.

Reply:
Thank you! :)